runtime/pom.xml

```diff
@@ -246,21 +246,22 @@
             <goals>
               <goal>unpack</goal>
             </goals>
+            <configuration>
+              <skip>${skipITs}</skip>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.keycloak</groupId>
+                  <artifactId>keycloak-server-dist</artifactId>
+                  <version>${keycloak.version}</version>
+                  <type>tar.gz</type>
+                  <outputDirectory>${project.build.directory}/keycloak</outputDirectory>
+                </artifactItem>
+              </artifactItems>
+            </configuration>
           </execution>
         </executions>
         <configuration>
-          <skip>${skipITs}</skip>
-          <artifactItems>
-            <artifactItem>
```

Comment:
Oh sorry you're right this snuck in and is unrelated to this PR... I moved the config into the unpack goal [above](https://github.com/syndesisio/syndesis-rest/pull/430/files#diff-48c560bbb8f61d82574c2887b65b498fR250) as `skipITs` for example shouldn't affect dependency analysis...